common/conf/env/DEV.properties

```diff
@@ -86,3 +86,6 @@ admin.oauth.callback=http://localhost:9000/oauth2callback
 #Preview Oauth
 preview.oauth.clientid=768784040766-1eq1drssgu65vrs4oi1hjh70d9b30fti.apps.googleusercontent.com
 preview.oauth.callback=http://localhost:9000/oauth2callback
+
+# Beacon diagnostics
+beacon.url=//localhost:9000
```

Comment:
I tend to do this as `127.0.0.1:9000` most people use `localhost:9000` as the url in their browser. That way it really does operate cross domain.